peekingduck/pipeline/nodes/input/live.py

```diff
@@ -24,7 +24,47 @@
 
 
 class Node(AbstractNode):
-    """Node to receive livestream as inputs."""
+    """Node to receive livestream as inputs.
+
+    Inputs:
+        None
+
+    Outputs:
+        |img|
+
+        |filename|
+
+        |pipeline_end|
+
+        |saved_video_fps|
+
+    Configs:
+        fps_saved_output_video (:obj:`int`): **default = 10**
+
+            FPS of the mp4 file after livestream is processed and exported.
+            FPS dependant on running machine performance.
+
+        filename (:obj:`str`):  **default = "webcamfeed.mp4"**
+
+            Filename of the mp4 file if livestream is exported.
+
+        resize (:obj:`Dict`): **default = { do_resizing: False, width: 1280, height: 720 }**
+
+            Dimension of extracted image frame
```

Comment:
use the same definition as `recorded.py`

Reply:
I will stick to "Dimension of extracted image frame" for both live and recorded